### PR TITLE
Feature/tri 569 create resource monitoring dashboards

### PR DESCRIPTION
**Pull request by ds-ext-ooyeneye**

### CODE REVIEW

**Comment by ds-jkreutzfeld** [comment]
I would like to avoid having separate dashboards for every environment. Please find a way to unify this.

**Comment by ds-jkreutzfeld** [comment]
Please also see the review comments by Martin in the Jira ticket :)

**Review by ds-jkreutzfeld — APPROVED** [review]

**Review by ds-jkreutzfeld — CHANGES_REQUESTED** [review]
Unfortunately, you broke the yaml files with the formatting.

{{ }} placeholders now look like this: { {  } }

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -65,5 +65,9 @@
     <codeStyleSettings language="kotlin">
       <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
     </codeStyleSettings>
+    <codeStyleSettings language="yaml">
+      <option name="SPACE_WITHIN_BRACKETS" value="false" />
+      <option name="SPACE_WITHIN_BRACES" value="false" />
+    </codeStyleSettings>
```
Comment (ds-jkreutzfeld) [inline .idea/codeStyles/Project.xml:71]: As this didn't help, we dont need the change.

**Review by ds-jkreutzfeld — APPROVED** [review]
Please revert the change in Project.xml, after that is can be merged.

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (ds-ext-ooyeneye) [reply .idea/codeStyles/Project.xml]: Has been reverted

**Pull request merged**